src/theme/components/Layout.tsx

```diff
@@ -47,6 +48,26 @@ const Layout = ({ children, title }: Props): JSX.Element => (
       <GlobalStyle />
       <Helmet title={title} />
       {children}
+      <CookieConsent
+        location="bottom"
+        cookieName={'ga-disable-G-D2D0FR22RN'}
```

Comment:
❓ duplicated value, same question as for the cookie removal

Reply:
Same answer 😉